proxy.conf.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
can you speak to the need for this proxy config?

Reply:
It's to address CORS errors to connect our servers. The proxy acts as a bridge between the client and the server because CORS was throwing a fit when we tried to interact with the server directly.